modules/vye/config/initializers/breakers.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/base'
+require 'common/client/configuration/rest'
+require 'breakers/statsd_plugin'
+
+# Not sure if any or all of these are needed
+require 'dgib/claimant_lookup/configuration'
+require 'dgib/claimant_status/configuration'
+require 'dgib/verification_record/configuration'
+require 'dgib/verify_claimant/configuration'
+
+Rails.application.reloader.to_prepare do
```

Comment:
Why is this service not added to the original breakers initializer? Matching redis namespaces can cause all the data in the other redis namespace to disappear.

Reply:
Based on our understanding, we added this to the original vets-api breakers initializer.  Good catch, thanks for bringing this to our attention, we're still learning our way around this.

Reply:
I understand. Thank you for making all these changes.

Reply:
This file needs to be removed. Breakers client should not be re-initialized.